README.with-advanced-features.md

```diff
@@ -129,11 +130,8 @@ echo <memonic> | dymd keys add $HUB_PERMISSIONED_KEY --recover
 sh scripts/settlement/register_rollapp_to_hub.sh
 ```
 
-### Register sequencer for rollapp on settlement
+*Note: it's a good idea to check DEPLOYER variable in the above script is correct*
```

Comment:
I suggest removing the deployer whitelist on the hub's scripts. as it was before.
the permissioned rollapp deployer is unneeded on local default run

Reply:
Thanks@mtsitrin 

I guess, this is the 'advanced features' tutorial, so we might want to include this, even if it's not strictly necessary?

Maybe I misunderstand

Thanks